dockstore-integration-testing/src/test/java/io/dockstore/webservice/ServiceIT.java

```diff
@@ -130,6 +133,54 @@ void checkWorkflowAndServiceHierarchy() {
         session.close();
     }
 
+    @Test
+    void checkServiceInCollection() {
+
+        final io.dockstore.openapi.client.ApiClient webClientAdminUser = getOpenAPIWebClient(ADMIN_USERNAME, testingPostgres);
```

Comment:
Don't need to change this, but should write these tests in a class only for OpenAPI, we have a couple examples. Reduces these package qualifications